kettle.js

```diff
@@ -1,37 +1,47 @@
 /*
-Kettle.
+Kettle Main Module Loader
 
 Copyright 2012-2013 OCAD University
+Copyright 2015 Raising the Floor (International)
 
 Licensed under the New BSD license. You may not use this file except in
 compliance with this License.
 
 You may obtain a copy of the License at
-https://github.com/gpii/kettle/LICENSE.txt
+https://github.com/fluid-project/kettle/blob/master/LICENSE.txt
 */
 
 var fluid = require("infusion"),
     kettle = fluid.registerNamespace("kettle");
 
-var loader = fluid.getLoader(__dirname);
-
-loader.require("./lib/utils.js");
+require("./lib/KettleUtils.js");
 
 fluid.module.register("kettle", __dirname, require);
 
-loader.require("./lib/app.js");
-loader.require("./lib/configLoader.js");
-loader.require("./lib/dataSource.js");
-loader.require("./lib/middleware.js");
-loader.require("./lib/request.js");
-loader.require("./lib/request.io.js");
-loader.require("./lib/server.js");
-loader.require("./lib/server.io.js");
-loader.require("./lib/session.js");
-loader.require("./lib/session.io.js");
+require("./lib/dataSource-core.js");
+require("./lib/dataSource-node.js");
+require("./lib/KettleApp.js");
+require("./lib/KettleConfigLoader.js");
+require("./lib/KettleMiddleware.js");
+require("./lib/KettleMultiConfig.js");
+require("./lib/KettleRouter.js");
+require("./lib/KettleRequest.js");
+require("./lib/KettleRequest.ws.js");
+require("./lib/KettleServer.js");
+require("./lib/KettleServer.ws.js");
+require("./lib/KettleSession.js");
 
 kettle.loadTestingSupport = function () {
     require("./lib/test/KettleTestUtils.js");
+    require("./lib/test/KettleTestUtils.http.js");
+    require("./lib/test/KettleTestUtils.ws.js");
+};
+
+// A variant of loadTestingSupport that does not complain if no tests are queued - for examples and samples
+kettle.loadTestingSupportQuiet = function () {
```

Comment:
This can't be included in a test case since it subverts part of our own testing infrastructure. I'll add an "istanbul ignore comment" ...